web/src/app/projects/repository/repository.component.ts

```diff
@@ -39,6 +39,9 @@ export class RepositoryComponent implements OnInit {
         if (this.repository.contributors) {
           this.sortingService.sortListByNumber(this.repository.contributors, 'total');
         }
+        if (this.repository.pullRequests) {
+          this.sortingService.sortListByDate(this.repository.pullRequests, 'createdOn');
```

Comment:
Can we add the generic types to to these method calls, even though the linter doesn't show an error